configure

```diff
@@ -343,18 +343,31 @@ def b(value):
 
 
 def pkg_config(pkg):
-  cmd = os.popen('pkg-config --libs %s' % pkg, 'r')
-  libs = cmd.readline().strip()
-  ret = cmd.close()
-  if (ret): return None
+  pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')
+  pkg_config_args = '--silence-errors'
+  try:
+    subprocess.check_output([pkg_config, '--version'])
```

Comment:
Is this explicit 'is pkg-config installed?' check necessary?  Perhaps you can fold it into the subprocess call below?  The net result - a `(None, None)` tuple - should be the same.